### ISSUE TITLE
Vehicle status.lastLocationUpdateTime is now seconds past epoch

### ISSUE THREAD

**Issue by barbeau** [OneBusAway/onebusaway-application-modules#177]
I got a report today from a user in Puget Sound that the "Last updated" time in the vehicle marker popup balloon is wrong in OBA Android:

![image](https://cloud.githubusercontent.com/assets/928045/15880011/56bdfbca-2cf7-11e6-8597-4ba204c7f4bc.png)

Here's a sample `trips-for-route` API request:
http://api.pugetsound.onebusaway.org/api/where/trips-for-route/1_102576.json?includeStatus=true&app_ver=62version=2&key=v1_BktoDJ2gJlu6nLM6LsT9H8IUbWc%3DcGF1bGN3YXR0c0BnbWFpbC5jb20%3D

...and a the `status` block from that response:

~~~
status: {
activeTripId: "1_30904954",
blockTripSequence: 12,
closestStop: "1_30570",
closestStopTimeOffset: -7,
distanceAlongTrip: 11774.797356666386,
frequency: null,
lastKnownDistanceAlongTrip: 0,
lastKnownLocation: {
lat: 47.58036422729492,
lon: -122.32905578613281
},
lastKnownOrientation: 0,
lastLocationUpdateTime: 1465348788,  // <-- Used to be milliseconds past epoch
lastUpdateTime: 1465348788000,
nextStop: "1_30590",
nextStopTimeOffset: 61,
orientation: 90,
phase: "in_progress",
position: {
lat: 47.58058676414876,
lon: -122.329063
},
predicted: true,
scheduleDeviation: 220,
scheduledDistanceAlongTrip: 11774.797356666386,
serviceDate: 1465282800000,
situationIds: [ ],
status: "default",
totalDistanceAlongTrip: 13179.505611838784,
vehicleId: "1_2710"
},
~~~

It looks like `lastLocationUpdateTime` went from being milliseconds past epoch (i.e., `System.currentTimeMillis()`) to seconds past epoch (i.e., POSIX time).

I'm guessing there was a server-side update recently that broke this?  From some quick testing it looks like it's broken for all routes I've tested (randomly grabbed 5-6) in Puget Sound and Washington, D.C.

Also, just a guess - GTFS-realtime timestamps are POSIX time, it's possible the GTFS-rt timestamp is getting passed through to the OBA REST API unaltered?

@sheldonabrown Would you be able to roll out a fix for this quickly?

**Comment by banshee** [comment]
FYI, I'm the user who reported it.  Feel free to ping me if you needed anything else.

**Comment by sheldonabrown** [comment]
Unfortunately I introduced this as part of:
https://github.com/OneBusAway/onebusaway-application-modules/issues/151

This commit fixes it:
https://github.com/camsys/onebusaway-application-modules/commit/e9dfd9dc599624dd0951beca0e6aefa0cefda5f1

The earliest I can roll this out is late next week -- it needs to go
through change management.  Is that sufficient?

Sheldon

**Comment by barbeau** [comment]
Yes, that should work.

**Comment by barbeau** [comment]
Thanks for the fix!

**Comment by sheldonabrown** [comment]
The fix rolled out this morning.

**Comment by barbeau** [comment]
Thanks!  From quick testing it looks like this is fixed in Puget Sound, but not WMATA in Washington, D.C.  Is that expected?

**Comment by sheldonabrown** [comment]
Yes, sorry about that.  Washington's fix will roll out Monday morning.

**Comment by barbeau** [comment]
Ok, thanks!

**Comment by sheldonabrown** [comment]
Washington rolled out.

**Comment by barbeau** [comment]
:+1: Do you want to close this issue now, or wait until https://github.com/camsys/onebusaway-application-modules/commit/e9dfd9dc599624dd0951beca0e6aefa0cefda5f1 is merged?

**Comment by sheldonabrown** [comment]
Good point!  I was hoping the merge to master would take care of this, but
that's stalled now on some build issues and code cleanup. (read:  I haven't
had time to get back to it).

I leave it to you -- I'm ok with keeping it open to remind us not to loose
the commit.

**Comment by barbeau** [comment]
I guess let's leave this open until the commit makes it here.
